src/scrapers/movie/IMDB.cpp

```diff
@@ -115,6 +116,15 @@ void IMDB::search(QString searchStr)
         new NetworkReplyWatcher(this, reply);
         connect(reply, &QNetworkReply::finished, this, &IMDB::onSearchIdFinished);
 
+    } else if (QVariant(includeAdult).toBool()) {
+        QUrl url = QUrl::fromEncoded(
+            QStringLiteral("https://www.imdb.com/search/title/?adult=include&view=simple&title=%1").arg(encodedSearch).toUtf8());
```

Comment:
Note: This URL will also include TV shows and other non-movies.

Reply:
Thank you for your bool suggestion. Tested and included.

Reply:
Do you think that the addition of "&title_type=feature,documentary,short,video" into the adult URL is better than nothing :-)?

Reply:
Yes, that should work, e.g. for The Simpsons: https://www.imdb.com/search/title/?adult=include&view=simple&title=Simpsons&title_type=feature,documentary,short,video

The series isn't listed. :)

Btw:

Reply:
TV shows and other non-movies filtered out (baded46).